autoinstall/RHEL/8/server_without_GUI/ks.cfg

```diff
@@ -30,14 +35,14 @@ user --name={{ new_user }} --password={{ vm_password_hash }} --groups=root,wheel
 sshkey --username={{ new_user }} "{{ ssh_public_key }}"
 {% endif %}
 
-# X Window System configuration information
-xconfig  --startxonboot
```

Comment:
Is this X Window system not needed?

Reply:
This options is for graphical login. This kickstart doesn't install desktop, so no graphical login.